<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceFormatter::ProjectConfigPage</class>
 <widget class="QWidget" name="SourceFormatter::ProjectConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>315</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" stretch="0,0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="kcfg_useDefault">
     <property name="text">
      <string comment="@option:check">&amp;Use default</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="generalGroupBox">
     <property name="title">
      <string comment="@title:group">General</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="kcfg_overrideKateIndentation">
        <property name="toolTip">
         <string comment="@info:tooltip">Override the editor indentation mode according to the formatting style for documents without a Kate modeline.</string>
        </property>
        <property name="text">
         <string comment="@option:check">Override editor indentation mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_modelinesEnabled">
        <property name="toolTip">
         <string comment="@info:tooltip">Add a Kate modeline according to the formatting style to formatted documents.</string>
        </property>
        <property name="text">
         <string comment="@option:check">Add Kate modeline</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="formattinStylesGroupBox">
     <property name="title">
      <string comment="@title:group">Formatting Styles</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="KDevelop::SourceFormatterSelectionEdit" name="formatterSelectionEdit" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDevelop::SourceFormatterSelectionEdit</class>
   <extends>QWidget</extends>
   <header>sourceformatterselectionedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
